src/mrpro/algorithms/optimizers/pgd.py

```diff
@@ -80,6 +81,11 @@ def pgd(
     backtrack_factor
         must be :math:`<=1`. if :math:`<1.`, backtracking rule for stepsize following https://www.ceremade.dauphine.fr/~carlier/FISTA
         is used
+    t_for_converging_solution
+        by default, the algorithm updates the variable t as described by Beck and Teboulle in the original FISTA paper.
+        If set to True, the algorithm updates t as suggested by Chambolle and Dossal https://inria.hal.science/hal-01060130v3/document
```

Comment:
make link work in docs.

see how references in docstring are done in other algorithms.